doc/index.rst

```diff
@@ -150,10 +150,10 @@ Optional environment variables
 
 Note that some of these currently tied only to a single option, but we still leave them for the future when more options become available (e.g. ament with BUILDER).
 
-* `ABICHECK` (default: false): If `true`, run a binary compatibility check with `ABICC <https://github.com/lvc/abi-compliance-checker>`_ and exit (see `below <#abi-checks>`_ for details).
-* `ABICHECK_FALLBACK` (default: not set): URL of archive to be used if no merge parent could be detected,
-* `ABICHECK_URL` (default: not set): URL of archive to check against. If empty, merge parent is used or `ABICHECK_FALLBACK` for regular commits.
-* `ABICHECK_VERSION` (default: not set): Readble version name. It will be read from the URL if possible.
+* `ABICHECK` (default: false): If `true`, run a binary compatibility check with `ABICC <https://github.com/lvc/abi-compliance-checker>`_ against the current branch (or the targeted branch in the pull/merge request), then exit (see `below <#abi-checks>`_ for details).
```

Comment:
It is not checked against branches, but either against an archive or the left-hand merge parents (= target branch of PRs).
This way it works for any Git repo and  even for tests ABI checks in forks.
What do you mean by "current branch"?